pkg/github/github_test.go

```diff
@@ -4,14 +4,44 @@ import (
 	"encoding/json"
 	"io/ioutil"
 	stdlibHttp "net/http"
+	"os"
+	"path/filepath"
 	"regexp"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
 	pkgHttp "github.com/cyberark/conjur-oss-suite-release/pkg/http"
+	"github.com/cyberark/conjur-oss-suite-release/pkg/repositories"
 )
 
+type MockClient struct {
+	*stdlibHttp.Client
+	AuthToken string
+}
+
+// NewClient creates a Client with an initialized parent stdlibHttp.Client
+// object
+func NewMockClient() *MockClient {
+	return &MockClient{
+		&stdlibHttp.Client{},
+		"",
+	}
+}
+
+func (client *MockClient) Get(url string) ([]byte, error) {
+	httpClient := generateHTTPClientWithFileSupportTransport()
+
+	if strings.Contains(url, "compare") {
+		return httpClient.Get("file://./testdata/compare_v3.json")
+	} else if strings.Contains(url, "CHANGELOG") {
+		return httpClient.Get("file://./testdata/simple_changelog.md")
+	}
+	// Return local data only
+	return httpClient.Get("file://./testdata/releases_v3.json")
+}
```

Comment:
Move the `type`, `NewMockClient`, and `Get` to the top of the file since they're not really tests but setups for a mock structure

Reply:
What do you mean top of the file, above the imports? For me they're already above the tests.

Reply:
Ah you're totally right - I didn't notice that there was a file break!